src/aequitas_cli/aequitas_audit.py

```diff
@@ -119,12 +120,35 @@ def audit(df, configs, model_id=1, preprocessed=False):
         bias_df = b.get_disparity_min_metric(groups_model)
     print('Any NaN?: ', bias_df.isnull().values.any())
     print('bias_df shape:', bias_df.shape)
+
+    aqp = Plotting()
```

Comment:
completely subjective, but I think it would make more sense, for me at least, for the class to be named `Plot`, since that's the thing you're constructing, (as opposed to "a plotting"). the module name can either match the class or remain general.

Reply:
LCH to update